test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -102,41 +120,85 @@ public void setup() throws Exception {
         this.controllerWrapper.awaitRunning();
         this.controller = controllerWrapper.getController();
 
-        //4. Create Scope
+        // 4. Create Scope
         this.controller.createScope(SCOPE).get();
         ClientConfig clientConfig = ClientConfig.builder().build();
         SocketConnectionFactoryImpl connectionFactory = new SocketConnectionFactoryImpl(clientConfig);
         this.connectionPool = new ConnectionPoolImpl(clientConfig, connectionFactory);
         this.keyValueTableFactory = new KeyValueTableFactoryImpl(SCOPE, this.controller, this.connectionPool);
+
+        // 5. Create segment helper
+        HostMonitorConfig hostMonitorConfig = HostMonitorConfigImpl.builder()
+                .hostMonitorEnabled(false)
+                .hostMonitorMinRebalanceInterval(10)
+                .containerCount(containerCount)
+                .hostContainerMap(HostMonitorConfigImpl.getHostContainerMap(ENDPOINT, servicePort, containerCount))
+                .build();
+        HostControllerStore hostStore = HostStoreFactory.createInMemoryStore(hostMonitorConfig);
+        executor = ExecutorServiceHelpers.newScheduledThreadPool(1, "collector");
+        segmentHelper = new SegmentHelper(connectionPool, hostStore, executor);
     }
 
 
     @After
     public void tearDown() throws Exception {
+        ExecutorServiceHelpers.shutdown(executor);
         this.controller.close();
         this.connectionPool.close();
         this.controllerWrapper.close();
         this.serverListener.close();
         this.serviceBuilder.close();
         this.zkTestServer.close();
+        this.segmentHelper.close();
     }
 
     /**
      * Smoke Test. Verify that the KeyValueTable can be created and listed.
      */
     @Test
     public void testCreateListKeyValueTable() {
+        // Verify default rolloverSizeBytes config.
+        val kvt0 = newKeyValueTableName();
```

Comment:
I don't understand the purpose of this test but I highly doubt it belongs here. This looks a lot like a controller unit test. Please write this unit test somewhere in the controller project. 

This class should have little to no changes as a result of your change. It's an integration test, not a unit test. Its purpose is to test the End-to-end functionality of KVTs without regards to the implementation details. The rollover size is an implementation detail. 

To test the client-side rolloversize, I suggest you look at `KeyValueTableImplTests` (in the client). You can then use the `MockController` there to verify that the correct rollover size is passed to the Controller.